### DISCUSSION TITLE
Install self-hosted

### DISCUSSION THREAD

**Discussion by owerrride** [outline/outline#5568] [Self Hosting]
Hello, i dont understand its bug or no, but look this plz.

Among install i got this mistake
![1](https://github.com/outline/outline/assets/115143645/181c7f61-264a-414b-970b-e38d877adc70)
![2](https://github.com/outline/outline/assets/115143645/41556b14-9404-4de1-b005-23b068d25e55)

**Comment by camflan** [comment]
I'm pretty sure you need to use an dotenv style config, not json.
https://www.npmjs.com/package/dotenv#%EF%B8%8F-usage

**Comment by willie-lin** [comment]
II'm also experiencing this problem and haven't found a solution yet。

**Comment by willie-lin** [comment]
`[+] Running 1/0
 ✔ Container outlinewiki-storage-1  Started                                                                                                                                                    0.0s 
yarn run v1.22.19
$ sequelize db:create --env=production-ssl-disabled

Sequelize CLI [Node: 18.16.1, CLI: 6.6.1, ORM: 6.29.0]

Loaded configuration file "server/config/database.json".
Using environment "production-ssl-disabled".

ERROR: connect ECONNREFUSED 127.0.0.1:5432

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

**Comment by hiepxanh** [comment]
I have the same issue, did anyone else can run it can connect to database?

**Reply by lumos0** [reply]
#5411
